src/databricks/labs/remorph/reconcile/constants.py

```diff
@@ -60,11 +63,36 @@ class ReportType(AutoName):
 class ThresholdMode(AutoName):
     PERCENTILE = "percentile"
     ABSOLUTE = "absolute"
-
-
-class ThresholdMatchType(AutoName):
-    INTEGER = "integer"
-    TIMESTAMP = "timestamp"
+    NUMBER_ABSOLUTE = "number_absolute"
+    NUMBER_PERCENTILE = "number_percentile"
+    DATETIME = "datetime"
+
+
+class ThresholdSQLTemplate(AutoName):
+    SELECT_NUMBER_ABSOLUTE = """source.{column} as {column}_source, databricks.{column} 
+                                as {column}_databricks, case when (coalesce(source.{column},0) - coalesce(databricks.{column},0)) == 
+                                0 then "Match"
+                                when (coalesce(source.{column},0) - coalesce(databricks.{column},0)) between {lower_bound} 
+                                and {upper_bound} then "Warning" else "Failed" end as {column}_match """
+    SELECT_NUMBER_PERCENTILE = """source.{column} as {column}_source, databricks.{column} 
```

Comment:
bad design: can you put these to the file where they're used? code locality is very important - e.g. the majority of the code that does the same chunk of work have to be in the same file. We don't have to jump between files to view the definition of constants.

don't put these to enum, it makes little sense from maintainability perspective.

Reply:
My bad this is all stale code after sqlglot expressions are used. Removed the code.